### ISSUE TITLE
dark mode

### ISSUE THREAD

**Issue by adnjoo** [npm/documentation#839]
can add dark mode?

**Comment by lukekarrys** [comment]
Yes, this is a great idea! Here are the tasks I see as needed for this:

- [ ] Add dark mode toggle to UI
- [ ] Use `primer/react` auto mode to detect system dark mode
- [ ] Save dark mode setting to local storage on change
- [ ] Audit use of `<Theme>` components so `theme="light"` sets it back to the default

**Comment by laine-hallot** [comment]
@lukekarrys is there a color set defined somewhere that this could be based on? I wouldn't mind making a PR for this

**Comment by laine-hallot** [comment]
@lukekarrys @adnjoo I actually just went ahead and implemented the dark mode stuff. If you want to take a look the PR is up here https://github.com/npm/documentation/pull/1480
